src/index.js

```diff
@@ -17,18 +19,30 @@ const displayTodo = (arr) => {
 
     const checkBox = document.createElement('input');
     checkBox.type = 'checkbox';
+    checkBox.checked = e.completed;
 
     const label = document.createElement('label');
     label.textContent = e.description;
 
     const listMenuIcon = document.createElement('i');
     listMenuIcon.className = 'fas fa-ellipsis-v';
 
+    const tasks = { li, arr, actions };
+    checkBox.addEventListener('change', taskComplete.bind(null, tasks));
+
     li.appendChild(checkBox);
     li.appendChild(label);
     li.appendChild(listMenuIcon);
     ul.appendChild(li);
   });
 };
 
-displayTodo(todoArr);
\ No newline at end of file
+const actions = new LocalStorageActions();
+const localTodos = actions.get();
+
+if (localTodos.length === 0) {
+  actions.add(todoArr);
+  displayTodo(todoArr, actions);
```

Comment:
- [ ] OPTIONAL - Consider adding the drag and drop functionality to make the [project interactive](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=The%20user%20should%20be%20able%20to%20reorder%20the%20list%20by%20dragging%20each%20item%20and%20moving%20it%20into%20the%20desired%20position.).